blueoil/cmd/convert.py

```diff
@@ -45,7 +53,14 @@ def create_output_directory(output_root_dir, output_template_dir=None):
 
 
 def get_output_directories(output_roor_dir):
```

Comment:
I know that it might be out of scope, but `output_roor_dir` should be `output_root_dir`?

Reply:
@tkng Thank you!! I did not notice that!
I created an issue.
https://github.com/blue-oil/blueoil/issues/573